airgun/views/task.py

```diff
@@ -28,7 +28,7 @@ class TasksView(BaseLoggedInView, SearchableViewMixin):
         "//div[./button[@id='tasks-dashboard-time-period-dropdown']]"
     )
     table = SatTable(
-        ".//div[@id='tasks-table']/table",
+        ".//div[@class='tasks-table']",
```

Comment:
I'm pretty sure `Table` widget should match `<table>` element, so you want to retain this `/table` part.

Also, seems in latest snap entire table is wrapped in `<div>`? I think that wasn't there before and looks like a small bug. So make that `//table`.

Reply:
OK, // table works, /table did not.
The extra <div> is there in previous snap too.
Thank you

Reply:
@mirekdlugosz thanks a lot, that fixed the reading of the table in my testing.